src/parser/deserializer.ts

```diff
@@ -135,6 +137,7 @@ function deserializeObject(
   const promoteBuffers = options['promoteBuffers'] == null ? false : options['promoteBuffers'];
   const promoteLongs = options['promoteLongs'] == null ? true : options['promoteLongs'];
   const promoteValues = options['promoteValues'] == null ? true : options['promoteValues'];
+  const promoteUUIDs = options['promoteUUIDs'] == null ? false : options['promoteUUIDs'];
```

Comment:
```suggestion
  const promoteUUIDs = options.promoteUUIDs == null ? false : options.promoteUUIDs;
```
We don't need to use hard coded computed property access, the `.` operator works fine. We should clean up the other options too, but let's save that for another PR.

Reply:
sounds good!